lib/puppet-lint/plugins/check_classes/arrow_on_right_operand_line.rb

```diff
@@ -18,26 +18,20 @@ def check
 
   def fix(problem)
     token = problem[:token]
-    tokens.delete(token)
+    remove_token(token)
 
     # remove any excessive whitespace on the line
     temp_token = token.prev_code_token
     while (temp_token = temp_token.next_token)
```

Comment:
maybe use https://github.com/rodjek/puppet-lint/commit/af5293330ea057cacf7c22b7386467dcea32393c here, and possibly a couple other places in this pull?